sandbox/public/index.html

```diff
@@ -74,6 +74,23 @@
         console.log("Sandbox: Set customer IDs event has completed.");
       });
 
+      alloy("syncIdByUrl", {
```

Comment:
Should this be named `syncIdsByUrl` (plural)?

Reply:
Are we going to be providing other commands that are like `syncIdsBySomething`? If so, why are we making them separate commands? If not, then should this one be named something more generic, like `syncIds` or `manuallySyncIds` or something like that? Are URL-based IDs the only identities that can be synced "manually"? I'm having a hard time understanding the overall/long-term vision here.